### PR TITLE
A Turkish language file has been added. 

### PR DESCRIPTION
**Pull request by egemenkus**
Additionally, hardcoded values in some files have been made compatible with translations, and incorrect values have been corrected.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/kumawallet/extension/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe: Language 

## What is the current behavior?
Some values were not being retrieved from the translation files despite being defined,
while others were not defined at all.

Issue Number: N/A

## What is the new behavior?
Some values were not coming from translation files. These values have been corrected and added to all translation files. Turkish language files have been added, and necessary checks have been implemented.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/pages/send/components/CommonFormFields.tsx [hunk 1]
```diff
@@ -1,111 +1,112 @@
...
-                  setValue("amount", value || "0");
-                }}
-                allowedDecimalSeparators={["%"]}
-              />
+    return (
+        <>
+            <div className="mx-auto">
+                <div className="flex gap-2 justify-center items-end mb-4">
+                    <div className="px-2">
+                        <p className="mb-2 font-inter font-light">{t("from")}:</p>
```
Comment (egemenkus) [inline src/pages/send/components/CommonFormFields.tsx:59]: This value was present as 'From:' in all languages. This issue has been fixed.

File: src/pages/send/components/CommonFormFields.tsx [hunk 2]
```diff
@@ -1,111 +1,112 @@
...
+                        <SelectableChain selectedChain={getValues("from") || {}}/>
+                    </div>
+                    <TbChevronRight size={26} className="mb-2"/>
+                    <div className="px-2">
+                        <p className="mb-2 font-inter font-light">{t("to")}:</p>
```
Comment (egemenkus) [inline src/pages/send/components/CommonFormFields.tsx:64]: This value was present as 'To:' in all languages. This issue has been fixed.

**Review by egemenkus — COMMENTED** [review]
Comments have been added in some places for better understanding of the changes

**Comment by rubenguc** [comment]
LGTM, thank you for adding the Turkish language and solve the issue with "to:"

**Review by rubenguc — APPROVED** [review]
LGTM, thank you for adding the Turkish language and solve the issue with "to:"

**Review by 0xslipk — APPROVED** [review]
LGTM. Thanks for your contribution 😎

**Review by 0xslipk — APPROVED** [review]

**Pull request merged**